tcms/testruns/tests/test_views.py

```diff
@@ -824,3 +826,25 @@ def test_should_fail_when_try_to_change_status_without_permissions(self):
         self.assertRedirects(
             self.client.get(self.url, {'finished': 1}),
             reverse('tcms-login') + '?next=%s?finished=1' % self.url)
+
+
+class TestCreateTestUsers(AssignPermissionsToTestUser):
+
+    def test_with_change_status_to_finished(self):
+        initiate_user_with_default_setups(self.tester)
+        self.test_run = TestRunFactory(product_version=self.version,
+                                       plan=self.plan,
+                                       build=self.build,
+                                       manager=self.tester,
+                                       default_tester=self.tester)
```

Comment:
this is causing a runtime error, because `AttributeError: 'TestCreateTestUsers' object has no attribute 'version'`, same goes for `plan`